Tests/Samples/LottieFiles/loading_gradient_strokes.json

```diff
@@ -0,0 +1 @@
phic - G-Stroke"},{"ty":"tr","p":{"a":0,"k":[0,0],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Ellipse 1","np":2,"cix":2,"ix":1,"mn":"ADBE Vector Group"},{"ty":"tm","s":{"a":1,"k":[{"i":{"x":[0.5],"y":[1]},"o":{"x":[0.167],"y":[0]},"n":["0p5_1_0p167_0"],"t":-20,"s":[0],"e":[0]},{"i":{"x":[0.5],"y":[1]},"o":{"x":[0.5],"y":[0]},"n":["0p5_1_0p5_0"],"t":-10,"s":[0],"e":[99]},{"t":61}],"x":"var $bm_rt;\n$bm_rt = loopOut('Cycle');","ix":1},"e":{"a":1,"k":[{"i":{"x":[0.5],"y":[1]},"o":{"x":[0.5],"y":[0]},"n":["0p5_1_0p5_0"],"t":-20,"s":[1],"e":[100]},{"i":{"x":[0.833],"y":[1]},"o":{"x":[0.5],"y":[0]},"n":["0p833_1_0p5_0"],"t":51,"s":[100],"e":[100]},{"t":61}],"x":"var $bm_rt;\n$bm_rt = loopOut('Cycle');","ix":2},"o":{"a":1,"k":[{"i":{"x":[0.5],"y":[1]},"o":{"x":[0.5],"y":[0]},"n":["0p5_1_0p5_0"],"t":-20,"s":[0],"e":[360]},{"t":61}],"x":"var $bm_rt;\n$bm_rt = loopOut('Cycle');","ix":3},"m":1,"ix":2,"nm":"Trim Paths 1","mn":"ADBE Vector Filter - Trim"}],"ip":-20,"op":160,"st":-20,"bm":0,"sr":1},{"ddd":0,"ind":4,"ty":4,"nm":"Shape Layer 1","ks":{"o":{"a":0,"k":100},"r":{"a":0,"k":0},"p":{"a":0,"k":[100,100,0]},"a":{"a":0,"k":[0,0,0]},"s":{"a":0,"k":[100,100,100]}},"ao":0,"shapes":[{"ty":"gr","it":[{"d":1,"ty":"el","s":{"a":0,"k":[150,150]},"p":{"a":0,"k":[0,0]},"nm":"Ellipse Path 1","mn":"ADBE Vector Shape - Ellipse"},{"ty":"gs","o":{"a":0,"k":100},"w":{"a":0,"k":10},"g":{"p":7,"k":{"a":0,"k":[0,0.918,0.263,0.208,0.167,0.951,0.5,0.114,0.333,0.984,0.737,0.02,0.5,0.594,0.698,0.173,0.666,0.204,0.659,0.325,0.833,0.231,0.59,0.641,1,0.259,0.522,0.957]}},"s":{"a":0,"k":[-57.125,-53.25]},"e":{"a":0,"k":[53.625,58.75]},"t":1,"lc":2,"lj":1,"ml":4,"nm":"Gradient Stroke 1","mn":"ADBE Vector Graphic - G-Stroke"},{"ty":"tr","p":{"a":0,"k":[0,0],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Ellipse 1","np":2,"cix":2,"ix":1,"mn":"ADBE Vector Group"},{"ty":"tm","s":{"a":1,"k":[{"i":{"x":[0.5],"y":[1]},"o":{"x":[0.167],"y":[0]},"n":["0p5_1_0p167_0"],"t":0,"s":[0],"e":[0]},{"i":{"x":[0.5],"y":[1]},"o":{"x":[0.5],"y":[0]},"n":["0p5_1_0p5_0"],"t":10,"s":[0],"e":[99]},{"t":81}],"x":"var $bm_rt;\n$bm_rt = loopOut('Cycle');","ix":1},"e":{"a":1,"k":[{"i":{"x":[0.5],"y":[1]},"o":{"x":[0.5],"y":[0]},"n":["0p5_1_0p5_0"],"t":0,"s":[1],"e":[100]},{"i":{"x":[0.833],"y":[1]},"o":{"x":[0.5],"y":[0]},"n":["0p833_1_0p5_0"],"t":71,"s":[100],"e":[100]},{"t":81}],"x":"var $bm_rt;\n$bm_rt = loopOut('Cycle');","ix":2},"o":{"a":1,"k":[{"i":{"x":[0.5],"y":[1]},"o":{"x":[0.5],"y":[0]},"n":["0p5_1_0p5_0"],"t":0,"s":[0],"e":[360]},{"t":81}],"x":"var $bm_rt;\n$bm_rt = loopOut('Cycle');","ix":3},"m":1,"ix":2,"nm":"Trim Paths 1","mn":"ADBE Vector Filter - Trim"}],"ip":0,"op":180,"st":0,"bm":0,"sr":1}]}
```

Comment:
Added an animation from LottieFiles that uses `GradientStroke` to make sure we have a test for this in the public regression suite